package.json

```diff
@@ -13,7 +13,10 @@
   "scripts": {
     "start": "node build/server.js",
     "test": "cake --use-js tests",
-    "build": "cake build"
+    "build": "cake build",
+    "lint": "npm run lint1 || npm run lint2",
```

Comment:
Yes. I use `||` so the client lint will run even if there are errors on server lint.
Depend if we want error in linting prevent next step (for example in build), or only use this command for informative purpose.